hw/dv/tools/xcelium/common.ccf

```diff
@@ -48,7 +48,8 @@ set_fsm_reset_scoring
 select_functional  -ams_control  -imm_asrt_class_package
 
 // Improve the scoping and naming of covergroup instances.
-set_covergroup -new_instance_reporting
+// including this added scoring of cip group and others as individual groups
+//set_covergroup -new_instance_reporting
```

Comment:
from the comment, looks like you wanted to enable this, but you actually commented this out

Reply:
@weicaiyang no I just forgot to delete this
I have cleaned it up.